### PR TITLE
DOC: actually produce docs for as_strided

### PR DESCRIPTION
**Pull request by eric-wieser**

### CODE REVIEW

**Comment by mhvk** [comment]
`as_strided` is not in the top-level name space (somewhat on purpose, I think). Given that, will this be in the right place?

**Comment by eric-wieser** [comment]
Good point. I guess this is a type of local build I can test, since it doesn't depend on the python source tree

File: doc/source/reference/routines.indexing.rst [hunk 1]
```diff
@@ -41,6 +41,7 @@ Indexing-like operations
    diag
    diagonal
    select
+   as_strided
```
Comment (shoyer) [inline doc/source/reference/routines.indexing.rst]: I think this needs to be `lib.stride_tricks.as_strided`.

But anyways, yes, you can definitely test this with a local docs build (`cd doc` followed by `make html`).

Comment (eric-wieser) [inline doc/source/reference/routines.indexing.rst]: `make html` doesn't work on windows, because `mkdir -p` means something different

Comment (eric-wieser) [inline doc/source/reference/routines.indexing.rst]: But `sphinx-build` works fine - and your suggestion is correct. Fixed!

Comment (shoyer) [inline doc/source/reference/routines.indexing.rst]: Ah, that's a bummer. I tried configuring a doc build on ReadTheDocs (see readthedocs.org/projects/numpy) but have not been able to get numpy building yet. I'm glad you figured out another way :)

**Comment by charris** [comment]
Works for me. Thanks Eric.

**Comment by eric-wieser** [comment]
Out of curiousity - is there a cronjob to produce https://docs.scipy.org/doc/numpy-dev/, or is that a manual endeavor? Would be good to have someway to sanity check formatting before a release, even if it's after a PR merge

**Comment by charris** [comment]
@eric-wieser Scipy has infrastructure for checking documentation for coverage, but I don't know if it allows viewing the results. We should take a look at it in any case, I believe it is the CircleCI test.

@rgommers Comment?

**Pull request merged**